packages/EasyActivity/src/Bridge/Symfony/Messenger/ActivityLogEntryMessage.php

```diff
@@ -5,9 +5,16 @@
 namespace EonX\EasyActivity\Bridge\Symfony\Messenger;
 
 use EonX\EasyActivity\ActivityLogEntry;
+use EonX\EasyLock\Interfaces\LockDataInterface;
+use EonX\EasyLock\Interfaces\WithLockDataInterface;
+use EonX\EasyLock\LockData;
 
-final class ActivityLogEntryMessage
+final class ActivityLogEntryMessage implements WithLockDataInterface
```

Comment:
will that fail if EasyLock isn't installed?

Reply:
Yes, the following error occurs without EasyLock:
```
Error: Interface "EonX\EasyLock\Interfaces\WithLockDataInterface" not found
```
https://github.com/eonx-com/easy-monorepo/runs/8109257071?check_suite_focus=true#step:7:17.

Reply:
@natepage we suggest installing this package in `composer.json`, similar to `symfony/messenger`. A project can use the package without async logic or with its own implementation,

Reply:
@roman-eonx so this message isn't triggered by default? do we have like a config option to enable async?

Reply:
@natepage `\EonX\EasyActivity\Bridge\Symfony\Messenger\AsyncDispatcher` is the default implementation of `\EonX\EasyActivity\Interfaces\AsyncDispatcherInterface`. And `\EonX\EasyActivity\Logger\AsyncActivityLogger` is the default implementation of `\EonX\EasyActivity\Interfaces\ActivityLoggerInterface`. Both are set in the Symfony bridge config. So, this is the default configuration for a Symfony project. But a project can implement its own dispatcher without Messenger or without locking. Or even use synchronous dispatching.

Reply:
ok thank you for the explanation